scripts/reactor_deployment.py

```diff
@@ -8,6 +8,72 @@
 
 
 # # # # # # # # # # # # Constituent Functions # # # # # # # # # # #
+def pull_start_index(df, start_year):
+    """
+    This function takes in a dataframe and a start year and returns the
+    index of the start year.
+
+    Parameters
+    ----------
+    df : :class:`pandas.DataFrame`
+        The dataframe to pull the start year from.
+    start_year : int
+        The year to start the deployment of advanced reactors.
+
+    Returns
+    -------
+    start_index : int
+        The index of the start year.
+    """
+    start_index = df[df['Year'] == start_year].index[0]
+
+    return start_index
+
+
+def capacity_increase(df, base_col, rate, start_year, end_year):
+    """
+    This function takes in an increase rate, and creates a new column in the
+    dataframe populated with the increased capacity.
+
+    Parameters
+    ----------
+    df : :class:`pandas.DataFrame`
+        The dataframe to add the new column to.
+    base_col : str
+        The name of the column to use as the base capacity.
+        This column should contain the net capacity.
+    rate : float
+        The percentage of change in capacity year-to-year
+        (e.g. 1.01 for a 1% increase).
+    start_year : int
+        The year to start the increase.
+    end_year : int
+        The year to end the increase.
+
+    Returns
+    -------
+    df : :class:`pandas.DataFrame`
+        The dataframe with the new column added.
+    """
+    # Apply lambda function for the range from start_year to end_year
+    df.loc[start_year:end_year-1, f"{base_col} Inc {rate}"] = \
+        df.loc[start_year:end_year-1].apply(
+            lambda row: df.loc[start_year, base_col] *
+            (rate)**(row.name - start_year), axis=1
+        )
```

Comment:
```suggestion
    capacity_change = lambda row: df.loc[start_year, base_col] *
            (rate)**(row.name - start_year)
    df.loc[start_year:end_year-1, f"{base_col} Inc {rate}"] = \
        df.loc[start_year:end_year-1].apply(
           capacity_change, axis=1
        )
```

I'm not sure if i got the whitespace correct, but this will be more readable if you define the lambda function, first.

Reply:
That's definitely more readable! Thanks for the suggestion.

Reply:
You can tell pep8 (and pep8speaks) to ignore errors like that. It also doesnt like it if import statements arent at the top of the file — but sometimes you need to append a directory to path first.